cmd/spire-server/cli/healthcheck/healthcheck.go

```diff
@@ -17,82 +16,52 @@ func NewHealthCheckCommand() cli.Command {
 	return newHealthCheckCommand(common_cli.DefaultEnv)
 }
 
-func newHealthCheckCommand(env *common_cli.Env) *healthCheckCommand {
-	return &healthCheckCommand{
-		env:     env,
-		timeout: common_cli.DurationFlag(time.Second * 5),
-	}
+func newHealthCheckCommand(env *common_cli.Env) cli.Command {
+	return util.AdaptCommand(env, new(healthCheckCommand))
```

Comment:
As mentioned in the PR description, most of the changes in this file are to use the common command structure, so I didn't have to implement the flag deprecation handling in an extra spot.